.gitignore

```diff
@@ -1,3 +1,4 @@
 ipfs.s9pk
 image.tar
 scripts/embassy.js
+docker-images
```

Comment:
best practice you showed me was adding an empty line at the end of this file.

Reply:
already there.